hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/response/s3/bucket/S3BucketCreateResponse.java

```diff
@@ -59,19 +59,15 @@ public S3BucketCreateResponse(
   public void addToDBBatch(OMMetadataManager omMetadataManager,
```

Comment:
addToDBBatch should be marked as protected, as in base class it is marked private, and it has below javadoc.
```
  /**
   * Implement logic to add the response to batch. This function should be
   * called from checkAndUpdateDB only.
   * @param omMetadataManager
   * @param batchOperation
   * @throws IOException
   */
  protected abstract void addToDBBatch(OMMetadataManager omMetadataManager,
      BatchOperation batchOperation) throws IOException
```

Reply:
Done.